mlos_bench/mlos_bench/config/schemas/optimizers/mlos_core-optimizer-subschema.json

```diff
@@ -60,7 +60,20 @@
                         "minimum": 0,
                         "maximum": 1,
                         "example": 0.1
+                    },
+                    "output_directory": {
+                        "description": "Directory to store SMAC output files.",
+                        "type": ["string", "null"],
+                        "pattern": "^[a-zA-Z0-9_./-]+$",
+                        "example": "smac_output"
+                    },
+                    "run_name": {
+                        "description": "Name to give for the smac output (e.g., $experimentId). If null is used, SMAC will calculate a hash based name.",
+                        "type": ["string", "null"],
+                        "pattern": "^[a-zA-Z0-9_.-]+$",
+                        "example": "$experimentId"
```

Comment:
Note: this won't actually work just yet.  We don't do `$var` expansion on all places.  Will tackle that in a different PR.